src/docs/app-monitoring/Sysdig-Monitoring-Guidline-for-shared-services-and-apps.md

```diff
@@ -0,0 +1,276 @@
+
+- Update project contact details and other metadata;
+- Request that their project namespace be created in additional clusters;
+- Request other resources such as KeyCloak realms or Artifactory pull-through repositories; and
+- Receive management from the platform services team.
+
+More details about the Registry app and its workings can be found [here](https://github.com/bcgov/platform-services-registry/blob/master/docs/Whole-project-workflow.md).
+
+### Setting up SRE
+The scope of SRE encompasses the deployment, configuration, and monitoring of the app. It also covers the availability, latency, change management, emergency response, and capacity management of services in production. To ensure optimal performance and reliability, we use several tools and methodologies that align with SRE principles.
+
+
+#### SLA
+The client should be at the center of every aspect of your customer agreement. An incident may need fixing ten different issues on the back end. But from the client's perspective, the only thing that matters is that the system works as it should, and facts should be reflected in your SLAs and SLOs. Be sure to limit your promises to high-level, user-facing functionality, and always use plain language in SLAs. Based on this, we can come up with some SLA for the Registry:
+
+* Normal users can successfully load the Registry dashboard within 5 seconds.
+* Admin user can successfully load Registry dashboard within 13 seconds.
+* An approved product request can be provisioned within an hour.
+* Application is up online 99% of the time.
+* Update product requests can be processed within an hour.
+
+
+#### SLO 
+The definition of s SLO is a service level objective: a target value or range of values for a service level that is measured by an SLI. Google has a really [good doc](https://sre.google/workbook/implementing-slos/#:~:text=For%20example%2C%20if%20you%20have,50%25%20of%20the%20error%20budget.) for how to gets the start.
+what you want to promise your customers is deciding how reliable you want your service to depend on what your customers expect. For example, if your SLA states that your customers will receive a response to each request they make in 300 milliseconds, then perhaps your SLO should state that the response will be returned in 200 milliseconds. Choosing an appropriate SLO is hard. Once we have SLA that we know can make our users happy, SLO is like the bottom line of our promise. Therefore, it is in our best interest to catch an issue before it beaches our SLA so that you have time to fix it. And this promise often comes up with consequences if we break it. Again, I will use the Registry as an example and the period is for each month:
+
+
+* Retrieving all product information from DB should be less than 8 sec. 
+* Retrieving 30 product information from DB should be less than 2 sec. 
+* Web, API, and DB should be up 99.5% of the time
+* DB should have a backup every 30 mins
```

Comment:
can you explain why a backup every 30 mins, for example which SLA does this match to?